modules/executors/variables.tf

```diff
@@ -188,3 +188,9 @@ variable "randomize_resource_names" {
   type        = bool
   description = "Use randomized names for resources. Disable if you are upgrading existing executors that were deployed using the legacy naming conventions, unless you want to recreate executor resources on GCP."
 }
+
+variable "use_local_ssd" {
```

Comment:
this requires that you use an instance type that allows to mount a single NVMe SSD - not all instances do that. Some need more than one.

Reply:
we could theoretically support that, too and raid0 them into a larger one.